apps/site/snippets/en/download/fnm.bash

```diff
@@ -1,14 +1,11 @@
-# installs fnm (Fast Node Manager)
-curl -fsSL https://fnm.vercel.app/install | bash
+# Download and install fnm:
+${props.os === 'WIN' ?
+  'winget install Schniz.fnm' :
+  'curl -o- https://fnm.vercel.app/install | bash'
+}
 
-# activate fnm
```

Comment:
this seems to be missing - [previously](https://nodejs-hj4exhlax-openjs.vercel.app/en/download/package-manager) fnm on mac/linux has the bashrc sourcing here, and windows has `fnm env --use-on-cd | Out-String | Invoke-Expression`.

Reply:
I tested without the sourcing and it worked for my environment at least?